libsodium-sys/build.rs

```diff
@@ -97,12 +96,18 @@ This function will set `cargo` flags.
 #[cfg(target_env = "msvc")]
 fn find_libsodium_pkg() {
     match vcpkg::probe_package("libsodium") {
-        Ok(_) => {
+        Ok(lib) => {
             println!(
                 "cargo:warning=Using unknown libsodium version.  This crate is tested against \
                  {} and may not be fully compatible with other versions.",
                 VERSION
             );
+            for lib_dir in &lib.link_paths {
+                println!("cargo:lib={}", lib_dir.to_string_lossy());
```

Comment:
You will silently get an invalid path in case there is an invalid unicode char in the path. See https://doc.rust-lang.org/nightly/std/path/struct.PathBuf.html#method.to_string_lossy . I believe you should use https://doc.rust-lang.org/nightly/std/path/struct.PathBuf.html#method.to_str and panic if it returns None (invalid unicode).

Reply:
But is it actually possible to get a unicode char in these situations?

Reply:
I'll make the modification, but I don't think it is possible since we don't directly get user input in these cases.